lib/ui/profile/components/profile_view.dart

```diff
@@ -18,73 +19,81 @@ class ProfileView extends StatelessWidget {
       withGradient: true,
       child: BlocBuilder<ProfileBloc, ProfileState>(
         builder: (context, state) {
-          return Scaffold(
-            backgroundColor: HyphaColors.transparent,
-            body: Stack(
-              children: [
-                const HyphaHalfBackground(showTopBar: false),
-                SingleChildScrollView(
-                  child: Padding(
-                    padding: const EdgeInsets.all(26),
-                    child: Column(
-                      crossAxisAlignment: CrossAxisAlignment.center,
-                      children: [
-                        const SizedBox(height: 80),
-                        HyphaEditableAvatarImage(
-                          imageRadius: 50,
-                          name: state.profileData?.name,
-                          imageFromUrl: state.profileData?.image,
-                          // imageFromFile: _file?.path,
-                          onImageRemoved: () {
-                            // setState(() {
-                            //   _file = null;
-                            // });
-                          },
-                          onImageSelected: (image) async {
-                            // setState(() {
-                            //   _file = image;
-                            // });
-                          },
+          return RefreshIndicator(
+            onRefresh: () async {
+              context.read<ProfileBloc>().add(const ProfileEvent.initial());
```

Comment:
well that's somewhat cheating. 

If "Initial" changes to something else that we won't want to happen then this breaks.